server/tpm2.go

```diff
@@ -67,7 +67,7 @@ func TPM2_LoadSRK(rwc io.ReadWriteCloser) (tpmutil.Handle, error) {
 	Seals the given data to the TPM Storage Root Key (SRK) and
 	returns the resulting private and public blobs
 */
-func TPM2_Seal(data []byte) ([]byte, []byte, error) {
+func TPM2_Seal(data []byte, pin string) ([]byte, []byte, error) {
```

Comment:
Please document the new parameter. In particular, this function has a `pin` parameter but also looks at --with-pin: how are the two related?